README.md

```diff
@@ -26,59 +26,67 @@ in the local environment.
 
 ### Deploy locally with Terraform
 
-Create a target Juju model:
+First, create Juju models for the Identity Platform and its dependencies:
 
 ```shell
-juju add-model <juju model>
+juju add-model identity-platform
+juju add-model core
 ```
 
-Make sure two ingresses (e.g. `traefik-k8s`) are deployed in the model, and the
-external ingress provides a Juju offer:
+#### Dependencies
 
-```shell
-# Deploy external ingress
-juju deploy traefik-k8s <external ingress app> --trust --channel <channel>
-
-# Deploy internal ingress
-juju deploy traefik-k8s <internal ingress app> --trust --channel <channel>
+Deploy the dependencies: [traefik](https://charmhub.io/traefik-k8s), [postgresql](https://charmhub.io/postgresql-k8s), [openfga](https://charmhub.io/openfga-k8s),
+and a certificates charm (e.g. [lego](https://charmhub.io/lego) or [self-signed-certificates](https://charmhub.io/self-signed-certificates)).
+And make sure they provide Juju offers:
 
-# Create the juju offer
-juju offer <external ingress app>:ingress <offer name>
+```shell
+# Deploy dependencies
+juju deploy traefik-k8s traefik-public --trust --channel latest/stable
+juju deploy postgresql-k8s --trust --channel 14/stable
+juju deploy openfga-k8s --trust --channel latest/stable
```

Comment:
I think you need to integrate openfga with postgres, otherwise it will go into a blocked state